XLS-0033d-compact-fungible-tokens/README.md

```diff
@@ -206,122 +216,28 @@ A set of flags indicating properties or other options associated with this **`CF
 | `lsfCFTLocked`       | `0x0001`   | If set, indicates that the CFT owned by this account is currently locked and cannot be used in any XRP transactions other than sending value back to the issuer. When this flag is set, the `LockedAmount` must equal the `CFTAmount` value. |
 | `lsfCFTAuthorized`       | `0x0002`   | (Only applicable for allow-listing) If set, indicates that the issuer has authorized the holder for the CFT. This flag can be set using a `CFTokenAuthorize` transaction; it can also be "un-set" using a `CFTokenAuthorize` transaction specifying the `tfCFTUnauthorize` flag. |
 
-##### 1.2.1.2.2. Example CFToken JSON
-
- ```json
- {
-     "TokenID": "00070C4495F14B0E44F78A264E41713C64B5F89242540EE255534400000000000000",
-     "Flags": 0,
-     "CFTAmount": "100000000",
-     "LockedAmount": "0"
- }
- ```
-
-#### 1.2.1.3. The **`CFTokenPage`** ledger entry
-
-This object represents a collection of **`CFToken`** objects owned by the same account. It is important to note that the **`CFToken`** objects themselves reside within this page, instead of in a dedicated object entry in the `SHAMap`. An account can have multiple **`CFTokenPage`** ledger objects, which form a doubly-linked list (DLL).
-
-In the interest of minimizing the size of a page and optimizing storage, the `Owner` field is not present since it is encoded as part of the object's ledger identifier (more details in the **`CFTokenPageID`** discussion below).
-
-##### 1.2.1.3.1 Fields
-
-A **`CFTokenPage`** object may have the following required and optional fields:
-
-| Field Name          | Required? | JSON Type | Internal Type |
-| ------------------- |-----------| --------- |---------------|
-| `LedgerEntryType`   | ✔️         | `string`  | `UINT16`      |
-| `PreviousPageMin`   | ️          | `string`  | `UINT256`     |
-| `NextPageMin`       | ️          | `string`  | `UINT256`     |
-| `PreviousTxnID`     | ️          | `string`  | `HASH256`     |
-| `PreviousTxnLgrSeq` | ️          | `number`  | `UINT32`      |
-| `CFTokens`          | ️✔         | `object`   | `TOKEN`      |
-
-###### 1.2.1.3.1.1. `**LedgerEntryType**` 
-
-Identifies the type of ledger object. This proposal recommends the value `0x007F` as the reserved ledger entry type.
-
-###### 1.2.1.3.1.2. `**PreviousPageMin**` 
-
-The locator of the previous page, if any. Details about this field and how it should be used are outlined below, after the construction of the **`CFTokenPageID`** is explained.
-
-###### 1.2.1.3.1.3. `**NextPageMin**` 
-
-The locator of the next page, if any. Details about this field and how it should be used are outlined below, after the construction of the **`CFTokenPageID`** is explained.
-
-###### 1.2.1.3.1.4. `**PreviousTxnID**` 
+###### 1.2.1.2.1.7. `OwnerNode`
```

Comment:
We need to add these two fields to the `CFToken` fields table:

```
|PreviousTxnID | ️ | string | HASH256
|PreviousTxnLgrSeq | ️ | number | UINT32
```

For definitions, we need to add this:

```
1.2.1.3.1.4. **PreviousTxnID**
Identifies the transaction ID of the transaction that most recently modified this CFToken object.

1.2.1.3.1.5. **PreviousTxnLgrSeq**
The sequence of the ledger that contains the transaction that most recently modified this CFToken object.
```

We should also add these fields to `CFTokenIssuance`. See https://github.com/XRPLF/rippled/pull/4751 for more details around a likely future amendment requiring these fields on all objects.

Reply:
I'll propose this in a separate PR.